src/apps/service/apps/combined/jobs.clj

```diff
@@ -32,8 +28,7 @@
        (partition-by app-step-partitioner)
        (map first)
        (map (fn [n step] (assoc step :step_number n))
-            (iterate inc 1))
-       (map add-job-step-type)))
```

Comment:
The query now includes the job step type, so this transformation is no longer required.